sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -512,6 +513,25 @@ def __init__(
             registry_reference=registry_reference,
             **app_insights_handler_kwargs,  # type: ignore[arg-type]
         )
+        # Evaluators
+        self._evaluators = EvaluatorOperations(
```

Comment:
so there will be no `_ai_client`? gen ai control plane will be merged into existing `_ml_client`?

Reply:
In the short term we will use the Model API to store evaluators, however there are plans to introduce the new service which will handle evaluator.